### ISSUE TITLE
Improve the error message when Certbot fails to install a certificate

### ISSUE THREAD

**Issue by bmw** [certbot/certbot#8232]
If you execute `certbot run` (or `certbot` without a subcommand) and it manages to obtain your certificate but not install it, we should help guide you on how to fix the problem. The idea here is to suggest a command they can run like `certbot install --cert-name <cert name>` to try to install the certificate again. While doing this, I think we should minimize obscuring the error output which they'll probably need to fix before that command will work though.

**Comment by bmw** [comment]
As flagged by Alex, we may actually want to do this near the end of the UX work since this message will currently often get hidden by the "IMPORTANT NOTES" section (and we don't want to add this new message to "IMPORTANT NOTES" since we're trying to remove that system entirely.)

**Comment by bmw** [comment]
For the sake of making the list of issues in our current milestone a little more attainable, I'm moving all of the issues in the UX project that are currently blocked to the next milestone, but feel free to recategorize it if you disagree.

**Comment by jeffsilverm** [comment]
I encountered a confusing error message, and while investigating that, I found another confusing error message.
````
2021-03-17 06:28:14,756:DEBUG:certbot.log:Exiting abnormally:
Traceback (most recent call last):
  File "/usr/bin/certbot", line 11, in <module>
    load_entry_point('certbot==0.31.0', 'console_scripts', 'certbot')()
  File "/usr/lib/python3/dist-packages/certbot/main.py", line 1365, in main
    return config.func(config, plugins)
  File "/usr/lib/python3/dist-packages/certbot/main.py", line 1119, in run
    certname, lineage)
  File "/usr/lib/python3/dist-packages/certbot/main.py", line 121, in _get_and_save_cert
    lineage = le_client.obtain_and_enroll_certificate(domains, certname)
  File "/usr/lib/python3/dist-packages/certbot/client.py", line 428, in obtain_and_enroll_certificate
    self.config)
  File "/usr/lib/python3/dist-packages/certbot/storage.py", line 1008, in new_lineage
    "live directory exists for " + lineagename)
certbot.errors.CertStorageError: live directory exists for jeffsilverm.ddns.net
(END)
````

This is confusing because it says that the live directory exists (which is true) but doesn't state where the live directory is or what should be done.

I inspected the code and found 2 similar error messages in `storage.py` and fixed them both.

````
root@raspberrypi:/usr/lib/python3/dist-packages/certbot# diff -y --suppress-common-lines --width=250  storage.py storage.py_ORIGINAL
                "archive directory " + archive +" exists for " + lineagename + ". Save its contents and get rid of the arc  |	                "archive directory exists for " + lineagename)
                "live directory " + live_dir + " exists for " + lineagename + ".  Save its contents and get rid of it")	    |	                "live directory exists for " + lineagename)
															    <
root@raspberrypi:/usr/lib/python3/dist-packages/certbot# diff  storage.py storage.py_ORIGINAL
1004c1004
<                 "archive directory " + archive +" exists for " + lineagename + ". Save its contents and get rid of the archive directory" )
---
>                 "archive directory exists for " + lineagename)
1008,1009c1008
<                 "live directory " + live_dir + " exists for " + lineagename + ".  Save its contents and get rid of it")
< 
---
>                 "live directory exists for " + lineagename)
root@raspberrypi:/usr/lib/python3/dist-packages/certbot# 

````

The next time I encountered the error, I get a much clearer error message:

````
root@raspberrypi:~# certbot run --nginx -d jeffsilverman.ddns.net
Saving debug log to /var/log/letsencrypt/letsencrypt.log
Plugins selected: Authenticator nginx, Installer nginx
Obtaining a new certificate
Performing the following challenges:
http-01 challenge for jeffsilverman.ddns.net
Waiting for verification...
Cleaning up challenges
live directory /etc/letsencrypt/live/jeffsilverman.ddns.net exists for jeffsilverman.ddns.net.  Save its contents and get rid of it
root@raspberrypi:~# 
````
'm not sure if my problem goes here or if should be a new issue.  I defer to your judgement.

**Comment by jsoref** [comment]
@jeffsilverm: could you please make a PR? At least then I'd have something I could use instead of reproducing your work.

**Comment by jeffsilverm** [comment]
Josh,

Josh:

Sure.  Take a look at PR 8807.

It's a minor fix.  However, since you are doing me this favor, I would like to gift you the following meme about small insignificant program changes: https://jeffsilverman.ddns.net/Humor_an_insignificant_software_change.png



Thank you,


Jeff

**Comment by ohemorange** [comment]
Closed by #8852.
